torchprime/torch_xla_models/experimental/custom_kernel.py

```diff
@@ -2,45 +2,14 @@
 import functools
 import json
 from dataclasses import asdict
-from typing import Any
 
 import torch
 import torch_xla.debug.profiler as xp
 from torch.library import custom_op
-from torch.utils._pytree import tree_flatten
-from torch_xla.core.xla_builder import jax_func_to_xla_computation
+from torch_xla.core.xla_builder import call_jax
 from torch_xla.distributed.spmd import Mesh
 from torch_xla.experimental.custom_kernel import requires_jax
 
-# use the global var to cache XlaComputation
-_FUNC_ARG_TO_XLA_COMPUTATION: dict[tuple[Any], str] = {}
-
-
-def call_jax(jax_func, args, kwargs=None, name=None):
```

Comment:
I think the current version of pinned torch_xla doesn't have those caching optimizations. Could you bump the pin e.g. replace `cxx11_20250313` with `20250402` everywhere?

This reminds me, it could be a good idea to test tracing time in E2E tests..